brayns/Brayns.cpp

```diff
@@ -376,41 +376,44 @@ struct Brayns::Impl : public PluginAPI
         auto& geometryParameters = _parametersManager.getGeometryParameters();
 
         registry.registerLoader(
-            {"MeshLoader", MeshLoader::getSupportedDataTypes(), [&] {
+            {MeshLoader::getName(), MeshLoader::getSupportedDataTypes(), [&] {
                  return std::make_unique<MeshLoader>(scene, geometryParameters);
              }});
```

Comment:
This is so repetitive that it makes sense to add a macro back, but one that reads

REGISTER_LOADER(Foo, ...)

A draft would be:

    #define REGISTER_LOADER(ClassName) \
        registry.registerLoader({#ClassName, ClassName::getSupportedDataTypes(), [&](){ \
                 return std::make_unique<ClassName>(scene, __VA_ARGS__); }})

You don't need getName with this macro